nuxeo-retention/src/main/resources/directories/retention_end.csv

```diff
@@ -1,3 +1,3 @@
-id,label_en,label_fr,obsolete
-"Document.Delete","Delete","Supprimer",0
-"Document.Trash","Trash","Mettre à la corbeille",0
+"id","label","obsolete","ordering"
```

Comment:
you need to define
```
"id","label","obsolete","ordering"
Document.Delete,"label.directories.retention_end.delete",0,0
Document.Trash,"label.directories.retention_end.trash",0,0
```
and then in 
```diff --git i/nuxeo-retention-web/i18n/messages.json w/nuxeo-retention-web/i18n/messages.json
index cdac195..955d7d7 100644
--- i/nuxeo-retention-web/i18n/messages.json
+++ w/nuxeo-retention-web/i18n/messages.json
@@ -11,6 +11,8 @@
   "eventType.afterSetRetention": "Retention period started",
   "eventType.retentionRuleAttached": "Retention rule applied",
   "label.document.type.retentionrule": "Retention Rule",
+  "label.directories.retention_end.delete": "Delete",
+  "label.directories.retention_end.trash": "Trash",
   "retention.action.dialog.heading": "Extend retention period",
   "retention.action.retain": "Extend retention",
   "retention.action.retain.extend": "Extend retention period",
```

Reply:
I tried before and didn't work, now I tried your suggestion let's see :)